_posts/2022-05-06-Setting-up-replication.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: How to set up replication from PostgreSQL to SAP HANA Cloud? 
+categories: [SAP BTP, SAP HANA Cloud, PostgreSQL]
+author_github: Kostya2702
+author: Konstantin Loshkarev
+---
+
+If you have a need to use new CAP technologies, and the database contains a lot of data, then you will need to replicate to SAP HANA Cloud. In this article, we will look at the complete setup using PostgreSQL without SSL as an example.
+
+## Database preconfiguration
+
+1) https://help.sap.com/docs/HANA_SMART_DATA_INTEGRATION/7952ef28a6914997abc01745fef1b607/f5e5dd27f7b1480ebdb392ce997b828a.html?locale=en-US
+
+    In `postgresql.conf` change the values ​​to the following:
+
+    ![Alt text](../article_images/database_preconfiguration1.png)
+    ![Alt text](../article_images/database_preconfiguration2.png)
+    ![Alt text](../article_images/database_preconfiguration3.png)
+
+    `max_replication_slots = database_to_replication * 2`
+
+2) Before registering the adapter, you need to install the database driver, information about this can be found at the bottom of the page (SAP HANA smart data integration and all its patches Product Availability Matrix (PAM) for SAP HANA SDI 2.0) by the address: 
+
+    https://help.sap.com/viewer/7952ef28a6914997abc01745fef1b607/2.0_SPS06/en-US/f8a15f52581e4f908c296fb92d0bec61.html
+
+3) Download the driver on the official website of the database. Then add the driver to the <DPAgent_root>/lib/ folder in .jar format
```

Comment:
Download the driver from the official website of the database and then add this driver to the <DPAgent_root>/lib/ folder using '.jar' format